_includes/configuration/contextmenu.md

```diff
@@ -7,16 +7,16 @@ The context menu option accepts three styles of item:
 * A `"|"` pipe character to indicate a separator should be added to delineate a group of menu items.
 * Context menu sections defined by a plugin (usually equal to the plugin name). Separators are automatically inserted between context menu sections.
 
-If the same name is registered as both a context menu section and a menu item, the section takes preference.
+If the same name is registered as both a context menu section and a menu item, the context menu section takes preference.
 
-The default configuration includes all plugins that provide a context menu; `link`, `image`, `imagetools`, `table`, and `spellchecker`.
+The default configuration includes the context menu items for all plugins that provide a context menu; `link`, `image`, `imagetools`, `table`, and `spellchecker`.
```

Comment:
This technically isn't true anymore, since we now include a `lists` section that isn't in the defaults, so not sure if you're worried?

Also probably should be `context menu sections` instead of `context menu items`?

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1495/commits/34f297c361e6e7ab7c3eb5b534e236056ed6fcf4